web_tool_script_2.R

```diff
@@ -24,6 +24,8 @@ set_portfolio_parameters(file_path = fs::path(par_file_path, paste0(portfolio_na
 
 set_project_parameters(file.path(working_location, "parameter_files", paste0("ProjectParameters_", project_code, ".yml")))
 
+data_location_ext <- sub("[0-9]{4}Q[0-9]", financial_timestamp, data_location_ext)
```

Comment:
```suggestion
data_location_ext <- sub("20[0-9]{2}Q[1-4]", financial_timestamp, data_location_ext)
```

see comment in `web_tool_script_1`

Reply:
fair enough 😄